src/components/Sidebar/components/NavItem/NavItem.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme }) => css`
+  label: nav-item;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  height: auto;
+  margin: ${theme.spacings.mega};
+  padding: ${theme.spacings.bit};
+  cursor: pointer;
+  color: ${theme.colors.n500};
+  fill: ${theme.colors.n500};
+`;
+
+const hoverStyles = ({ theme, selected }) => css`
+  label: nav-item__hover;
+  @media (hover: hover) {
```

Comment:
Out of curiosity, what's your reasoning for using a hover media query here? :)

Reply:
But why? Shouldn't users also get feedback to their actions when on a touchscreen device?